src/TMS/dev/HeatPump.cpp

```diff
@@ -0,0 +1,36 @@
+#include <TMS/dev/HeatPump.hpp>
+
+namespace TMS {
+
+HeatPump::HeatPump(IO::PWM& pwm, uint8_t initSpeed) : pwm(pwm) {
```

Comment:
For the `initSpeed` when the TMS system starts up, we do not want the pump running, so the pump should start out stopped.

Reply:
It's now been fully removed.